src/plugins/footnotes/index.ts

```diff
@@ -203,13 +214,32 @@ export default () => {
                 )
               )
             }
+
+            if (
+              tableInlineFootnoteIds &&
+              !tableInlineFootnoteIds.has(node.attrs.id)
+            ) {
+              decorations.push(
+                Decoration.widget(
+                  pos + node.nodeSize - 1,
+                  uncitedFootnoteWidget()
+                )
+              )
+            }
           }
           if (node.type === schema.nodes.footnotes_element) {
             decorations.push(
               Decoration.node(pos, pos + node.nodeSize, {
                 class: 'footnote-element',
               })
             )
+
+            if (parent?.type === schema.nodes.table_element_footer) {
+              tableInlineFootnoteIds = findTableInlineFootnoteIds(
+                node,
```

Comment:
You don't need to pass it, the position should be enough.

Reply:
updated